torchrec/csrc/dynamic_embedding/tensor_list.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+#include <torch/torch.h>
+
+namespace torchrec {
+
+class TensorList : public torch::CustomClassHolder {
```

Comment:
Looks like it's only used to pass a list of tensors from Python -> C++ . You should be able to pass a Python list of Tensors via pybind and then access them via pybind API.